### PR TITLE
ex1: microscopy coding done

### PR DESCRIPTION
**Pull request by annagav**
Completed the microscopy model for NSF Exercise 1.

### CODE REVIEW

File: html_app/master_model.js [hunk 1]
```diff
@@ -12631,13 +12631,10 @@ var __scb_sample_1 = {
...
                 disable_brightness: false
             },
-            western_blot: {format: "%CELL_LINE%, %TREATMENT%, %PP1% %TEMPERATURE%",
+            western_blot: {format: "%CELL_LINE%, %TREATMENT%",
                 keys: {
                     '%CELL_LINE%': {attr: ['cell_line'], map: ['cell_lines', '%KEY%', 'name']},
-                    '%TREATMENT%': {attr: ['treatment_list', 'list', '0', 'drug_list', 'list', '0', 'drug_id'], map: ['drugs', '%KEY%', 'name']},
-                    '%CONCENTRATION%': {attr: ['treatment_list', 'list', '0', 'drug_list', 'list', '0', 'concentration_id'], map: ['concentrations', '%KEY%', 'name']},
-                    '%TEMPERATURE%': {attr: ['treatment_list', 'list', '0', 'temperature'], map: ['experiment_temperatures', '%KEY%', 'name']},
-                    '%PP1%': {attr: ['treatment_list', 'list', '0', 'drug_list', 'list', '1', 'drug_id'], map: ['drugs', '%KEY%', 'short_name'], default: ''}
```
Comment (pdpinch) [inline html_app/master_model.js:12640]: Why were these lines removes?

Comment (annagav) [inline html_app/master_model.js:12640]: Because there is no temperature or concentration specified for this assignment.

**Comment by pdpinch** [comment]
This looks good to me @annagav 

Can this be merged before #209 ?

**Comment by annagav** [comment]
Yes, it can!

File: html_app/master_model.js [hunk 2]
```diff
@@ -12631,13 +12631,10 @@ var __scb_sample_1 = {
...
+                    '%TREATMENT%': {attr: ['treatment_list', 'list', '0', 'drug_list', 'list', '0', 'drug_id'], map: ['drugs', '%KEY%', 'name']}
```
Comment (carsongee) [inline html_app/master_model.js:12637]: nit: break into multiple lines

**Comment by annagav** [comment]
Updated!

**Comment by pdpinch** [comment]
@carsongee let me know when we can do a release to scbdev.mit.edu

**Comment by carsongee** [comment]
you mean scbdev.odl.mit.edu right?

**Comment by pdpinch** [comment]
yes, I mean scbdev.odl.mit.edu http://scbdev.odl.mit.edu/

**Comment by carsongee** [comment]
It is deployed

**Pull request merged**